notebook/nbextensions.py

```diff
@@ -831,7 +849,43 @@ class DisableNBExtensionApp(ToggleNBExtensionApp):
     Usage
         jupyter nbextension disable [--system|--sys-prefix]
     """
-    _toggle_value = None
+    version = __version__
+    flags = {"py" : ({
+        "DisableNBExtensionApp" : {
+            "python" : True,
+        }}, "Disable an extension from a Python package name")
+    }
+    flags['python'] = flags['py']
+    aliases = {'section': 'ToggleNBExtensionApp.section'}
```

Comment:
This should still inherit the --system, --user, --sys-prefix flags, etc. from `ToggleNBExtensionApp`. There's still a valid use case:

A system-wide installation enabled an extension, but you want to disable it in your own configuration. Passing `--user` in this case (which should store False, not pop config) makes sense as an override of the system defaults.